lib/form_data.js

```diff
@@ -76,6 +77,19 @@ FormData.prototype.append = function(field, value, options) {
 
   // pass along options.knownLength
   this._trackLength(header, value, options);
+
+  if (!this._entryList[field]) {
+    this._entryList[field] = [];
+  }
+  this._entryList[field].push(value);
```

Comment:
Append an object with the following shape

```
{ name: '', value: '' }
```

`append` do not remove previous entry, just do an `append` on the list.